spec/mixlib/log/formatter_spec.rb

```diff
@@ -17,7 +17,7 @@
 #
 
 require "time" unless defined?(Time.zone_offset)
-require File.expand_path(File.join(File.dirname(__FILE__), "..", "..", "spec_helper"))
+require File.expand_path(File.join(__dir__, "..", "..", "spec_helper"))
```

Comment:
```suggestion
require "spec_helper"
```

`spec/` is automatically on the load path so this is likely equivalent.